### ISSUE TITLE
OSError: [Errno 38] Function not implemented

### ISSUE THREAD

**Issue by Heath123** [Robert-N7/abmatt#24]
On Linux (Ubuntu 20.04):
```
Exporting to /run/user/1000/doc/6662a94c/luigi.dae...
Traceback (most recent call last):
  File "/home/heath/Downloads/abmatt/abmatt/gui/converter.py", line 67, in run
    self.item.convert()
  File "/home/heath/Downloads/abmatt/abmatt/converters/convert_lib.py", line 199, in convert
    self.save_model()
  File "/home/heath/Downloads/abmatt/abmatt/converters/convert_dae.py", line 53, in save_model
    self._end_saving(mesh)
OSError: [Errno 38] Function not implemented: 'luigi_maps'
```
`luigi` changes to whatever filename you saved as. I'll try to investigate to see what causes this

**Comment by Heath123** [comment]
So far I've found that it happens in `converter.batch_decode`, narrowing it down now

**Comment by Heath123** [comment]
Happens at https://github.com/Robert-N7/abmatt/blob/fbdfae8fb0a39cea79b26b4de38f8f16e7946bab/abmatt/image_converter.py#L312

**Comment by Heath123** [comment]
My guess is that running it in PyCharm in a snap is breaking it?

**Comment by Heath123** [comment]
I think I have a fix that I can PR

**Comment by Heath123** [comment]
`OSError: [Errno 38] Function not implemented: '/run/user/1000/doc/20b802ca/test_maps'`
Even absolute paths don't work?

**Comment by Heath123** [comment]
> My guess is that running it in PyCharm in a snap is breaking it?
> EDIT: Actually maybe not, still looking into it

It was this, running outside of a snap fixes it (why do I always leave so many comments on issues on this?)
